Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -274,7 +286,8 @@ public OrderTicket UpdateOrder(UpdateOrderRequest request)
         public OrderTicket CancelOrder(CancelOrderRequest request)
         {
             OrderTicket ticket;
-            if (!_orderTickets.TryGetValue(request.OrderId, out ticket))
+            if (!_openOrderTickets.TryGetValue(request.OrderId, out ticket) &&
+                !_closedOrderTickets.TryGetValue(request.OrderId, out ticket))
```

Comment:
Shouldn't `CancelOrder` only search open orders?

Reply:
I've replied completely in the first comment :)